nebari-dev/nebari
# Validate instance types for GCP

## Reference Issues or PRs

Closes https://github.com/nebari-dev/nebari/issues/2718

Adds back the `instances(region:str)` function to the GCP provider and validates available instances against the nebari config.



## What does this implement/fix?

_Put a `x` in the boxes that apply_


## Testing


## How to test this PR?



Check out and install this branch, then follow the instructions to [deploy nebari on GCP](https://www.nebari.dev/docs/how-tos/nebari-gcp/). Run `nebari validate -c nebari-config.yaml` to validate available instance types.

## Any other comments?
